lib/Connection.js

```diff
@@ -76,6 +76,11 @@ class Connection extends EventEmitter {
   }
 
   /**
+   * @param {function()} callback
+   */
+  setClosedCallback(callback) { this._closeCallback = callback; }
```

Comment:
can you please format this?
```js
setClosedCallback(callback) {
  this._closeCallback = callback;
}
```

Reply:
Done, so no inline on class method? Inline only on normal code?

Reply:
"Inlining" is not about code formatting.
To "inline a function" means to get the function body and paste it instead of function invocations.

More on wiki: [inline expanstion](https://en.wikipedia.org/wiki/Inline_expansion)